Documentation/tutorials/data-prep-tutorial.md

```diff
@@ -0,0 +1,210 @@
+# Data Prep for Data Collection <!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
+- [Overview](#overview)
+  - [Environment](#environment)
+  - [Prerequisites](#prerequisites)
+- [Data Prep for Data Collection mapping](#data-prep-for-data-collection-mapping)
+  - [Mapping custom `contextData` keys](#mapping-custom-contextdata-keys)
+  - [Mapping calculated fields using Analytics functions](#mapping-calculated-fields-using-analytics-functions)
+- [Validating Data Prep mapping using Assurance](#validating-data-prep-mapping-using-assurance)
+
+## Overview
+This tutorial covers how to use Data Prep for Data Collection to enable using Edge Bridge event data in other Experience Platform applications, by mapping event data to the [Experience Data Model (XDM)](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html) specification.
+
+```mermaid
+graph TD;
+    step1(1<br/>Existing app using Adobe Analytics) -->
+    step2(2<br/>Adobe Experience Platform<br/>Update server-side configuration) --> 
+    step3(3<br/>Edge Bridge<br/>Update app to send event data via Edge Network to Analytics) --> 
+    step4(4<br/>Assurance<br/>Verify implementation) -. "Optionally" .-> 
```

Comment:
- Shall we remove this optional step since this tutorial is dedicated for this workflow?
- Recommend using a success color for indicating this tutorial's focus, for example light green/blue, and faded gray for the non-covered steps? Likewise for the other tutorial